core/src/main/java/org/jsmart/zerocode/core/engine/tokens/ZeroCodeValueTokens.java

```diff
@@ -10,6 +10,7 @@
  */
 public class ZeroCodeValueTokens {
     public static final String JSON_PAYLOAD_FILE = "JSON.FILE:";
+    public static final String YML_PAYLOAD_FILE = "YML.FILE:";
```

Comment:
i think this is better to use the `YAML` syntax because this is the recommanded way: https://yaml.org/faq.html
```suggestion
    public static final String YAML_PAYLOAD_FILE = "YAML.FILE:";
```